R/debug.R

```diff
@@ -149,7 +149,11 @@ get_timestamp_diff <- function() {
 }
 
 get_timestamp_stamp <- function() {
-  paste0(format_date(Sys.time()), " ")
+  paste0(format_date(systime()), " ")
+}
+
+systime <- function() {
```

Comment:
It's probably not necessary to wrap this in a new function?

Reply:
This one might have actually been another situation that tricked me. I'll see if I can simplify this.